src/ProgressOnderwijsUtils/SchemaReflection/DatabaseDescription.cs

```diff
@@ -16,42 +16,69 @@ public enum DbObjectId { }
     [DbIdEnum]
     public enum DbColumnId { }
 
-    public struct DbNamedTableId : IWrittenImplicitly
+    public struct DbNamedObjectId : IWrittenImplicitly
     {
         public DbObjectId ObjectId { get; set; }
         public string QualifiedName { get; set; }
-
-        public static DbNamedTableId[] LoadAll(SqlConnection conn)
-            => SQL($@"
-                select
-                    ObjectId = t.object_id
-                    , QualifiedName = schema_name(t.schema_id) + N'.' + object_name(t.object_id)
-                from sys.tables t
-            ").ReadPocos<DbNamedTableId>(conn);

```

Comment:
Het is vrijwel zeker minder code als je deze versie aanhoud met een extra arg "type" en dan 2x querien, en dat is ook vast niet trager, maar heel boeiend is het niet.